HISTORY.rst

```diff
@@ -9,6 +9,7 @@ v0.2.3
 * Make the gsutil API consistent, so that we have `cp`, `sync` and `rm`, each of which
   accept the same args and kwargs (:issue:`69`)
 * Swap ``bumpversion`` for ``setuptools_scm`` to handle versioning (:issue:`78`)
+* Cast coordinates to dict before gathering in ``rhg_compute_tools.xarray.dataarrays_from_delayed`` and ``rhg_compute_tools.xarray.datasets_from_delayed``. This avoids a mysterious memory explosion on the local machine. Also add ``name`` in the metadata used by those functions so that the name of each dataarray or Variable is preserved.
```

Comment:
Oh! You're going to want to append an
```
(:issue: `83`)
```
to your line in this entry.

Reply:
nice catch! fixed